pack.sh

```diff
@@ -40,9 +40,10 @@ scripts/run-rosetta.sh --infuse --pkgs-from $TMPDIR/jsii.txt
 
 # Jsii packaging (all at once using jsii-pacmak)
 echo "Packaging jsii modules" >&2
+rosetta_cache_file=$HOME/.s3buildcache/rosetta-cache.tabl.json
 $PACMAK \
   --verbose \
-  --rosetta-tablet samples.tabl.json \
+  --rosetta-tablet ${rosetta_cache_file} \
```

Comment:
This should not be necessary anymore here.

Reply:
removed